docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -26,7 +26,7 @@ metrics based on the tracing data.
   * [Netflix Spring Cloud Feign](https://github.com/spring-cloud/spring-cloud-openfeign) 1.1.x -> 2.x
   * [Okhttp](https://github.com/square/okhttp) 2.x -> 3.x -> 4.x
   * [Apache httpcomponent HttpClient](http://hc.apache.org/) 2.0 -> 3.1, 4.2, 4.3, 5.0, 5.1
-  * [Spring RestTemplate](https://github.com/spring-projects/spring-framework) 4.x
+  * [Spring RestTemplate](https://github.com/spring-projects/spring-framework) 4.x -> 6.x
```

Comment:
RestTemplate 6 plugin is in opt1 optional too?

Reply:
> RestTemplate 6 plugin is in opt1 optional too?
Should be opt2 actually

> ¹Due to license incompatibilities/restrictions these plugins are hosted and released in 3rd part repository,
> go to [SkyAPM java plugin extension repository](https://github.com/SkyAPM/java-plugin-extensions) to get these.

> ²These plugins affect the performance or must be used under some conditions, from experiences. So only released in `/optional-plugins` or `/bootstrap-plugins`, copy to `/plugins` in order to make them work.